content/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -50,6 +50,18 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
 
 To prevent sending a massive number of requests, there are rate limits for method calls. After exceeding those limits, the requester will get a `Too many requests` error in the response of a specific request. After some time, the request will be unblocked so it can be sent again.
 
+## Pagination
+
+Pagination is a mechanism that allows splitting the database output into more manageable chunks of data.
+Based on the `limit` and `sort order` parameters, pagination is able to decide how many records will be returned at once and whether it should fetch the oldest or the latest data first.
+
+Any filters that could be applied should be provided in first pagination request. In the response, you'll get the `next_page_id` and `previous_page_id` parameters.
+You should make the subsequent request using one of these parameters as `page_id`, depending on the direction of iteration: forward or backward.
+
+The `filters`, `limit`, and `sort_order` parameters can't be provided along with `page_id`.
+
+The maximum duration of the `page_id` parameter before it expires is one month.
```

Comment:
I think it would be nice to wrap this information in some `notice`

Reply:
Good idea. We could use the :bulb: emoji (docs support it)